ios/RNGeth.swift

```diff
@@ -20,12 +20,39 @@ struct RuntimeError: LocalizedError {
     }
 }
 
+extension Data {
+    var bytes:[UInt8] { // fancy pretty call: myData.bytes -> [UInt8]
+        return [UInt8](self)
+    }
+
+    init?(hexString: String) {
```

Comment:
this feels dangerous without tests

Reply:
Valid point. I'll see if I can get rid of this. It mostly has to do with passing data around between the javascript and go layer. Things like hex strings turning into Byte Arrays and vice-versa. But there might be a cleaner way / with less code.